### ISSUE TITLE
dapr init fails in windows 11

### ISSUE THREAD

**Issue by tikluganguly** [dapr/cli#973]
Hi All,

          I am getting the following error when I am trying to do dapr init 

C:\Users\TikluGanguly>dapr init
Making the jump to hyperspace...
Container images will be pulled from Docker Hub
Installing runtime version 1.7.2
Downloading binaries and setting up components...
docker: Error response from daemon: container b77da94892ebb77bcf6ce868e0b21e8a1025a2826e8d8f93acc0bef806cd69dd encountered an error during hcsshim::System::CreateProcess: failure in a Windows system call: The system cannot find the file specified. (0x2)
[Event Detail:  Provider: 00000000-0000-0000-0000-000000000000]
[Event Detail:  Provider: 00000000-0000-0000-0000-000000000000]
[Event Detail: onecore\vm\compute\management\orchestration\vmhostedcontainer\processmanagement.cpp(173)\vmcomputeagent.exe!00007FF793019F4B: (caller: 00007FF792FCE13A) Exception(2) tid(388) 80070002 The system cannot find the file specified.
    CallContext:[\Bridge_ProcessMessage\VmHostedContainer_ExecuteProcess]
 Provider: 00000000-0000-0000-0000-000000000000].

Please guide

Thanks
Tiklu

**Comment by tikluganguly** [comment]
Here is the output of my docker version

>docker version

Client:
 Cloud integration: v1.0.23
 Version:           20.10.14
 API version:       1.41
 Go version:        go1.16.15
 Git commit:        a224086
 Built:             Thu Mar 24 01:53:11 2022
 OS/Arch:           windows/amd64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.7.1 (77678)
 Engine:
  Version:          20.10.14
  API version:      1.41 (minimum version 1.24)
  Go version:       go1.16.15
  Git commit:       87a90dc
  Built:            Thu Mar 24 01:48:25 2022
  OS/Arch:          windows/amd64
  Experimental:     true

**Comment by tikluganguly** [comment]
Ok, I was finally able to solve this issue. I was getting this issue as my container was using windows. so switched it to Linux first. 
![image](https://user-images.githubusercontent.com/222026/166914716-6974a9ce-9ef8-4e49-b4eb-63c0e658151d.png)

next I uninstalled dapr by typing 
dapr uninstall

there were still some residue container images and volume was present. I deleted them from docker desktop GUI

then again ran dapr init 

this does install all the images but got errors with ports. So ran the following command from a admin powershell
net stop winnat

after that started stopped the containers from the GUI 

then ran the following command to re ran winnat
 
net start winnat

now everything seems to be working fine
